transport/src/main/java/io/netty/channel/DefaultChannelPipeline.java

```diff
@@ -1188,11 +1188,14 @@ protected void onUnhandledInboundChannelInactive() {
      * in {@link ChannelInboundHandler#channelRead(ChannelHandlerContext, Object)}. This method is responsible
      * to call {@link ReferenceCountUtil#release(Object)} on the given msg at some point.
      */
-    protected void onUnhandledInboundMessage(Object msg) {
+    protected void onUnhandledInboundMessage(ChannelHandlerContext ctx, Object msg) {
```

Comment:
as this is public API we can not really do this as this is a breaking change :/

Reply:
I thought this - https://github.com/netty/netty/pull/7286#issuecomment-355434128 was an approval. Or I get it wrong?

Reply:
I am not sure if you got it wrong or if @carl-mastrangelo did not realise that it would be a breaking change. In any case we can not do this for 4.1. In master we can....

Reply:
Oh, I expected the single arg function to daisy chain to the multi arg one.  That's the typical pattern I thought.